lib/service/service.go

```diff
@@ -1584,6 +1590,15 @@ func (process *TeleportProcess) initSSH() error {
 			return trace.Wrap(err)
 		}
 
+		// asyncEmitter makes sure that sessions do not block
+		// in case if connections are slow
+		asyncEmitter, err = events.NewAsyncEmitter(events.AsyncEmitterConfig{
+			Inner: emitter,
+		})
+		if err != nil {
+			return trace.Wrap(err)
+		}
+
```

Comment:
should all these emitters get pulled out to `TeleportProcess` level?
or at least a helper function, so all the setup stays consistent across clients